mappings/net/minecraft/world/ModifiableWorld.mapping

```diff
@@ -44,10 +42,20 @@ CLASS net/minecraft/class_1945 net/minecraft/world/ModifiableWorld
 		COMMENT change persisted across loads. It's recommended to check whether this world is client before
 		COMMENT interacting with the world in this way.</p>
 		COMMENT
-		COMMENT <p>For the accepted values of the flags, see {@link net.fabricmc.yarn.constants.SetBlockStateFlags}.
+		COMMENT <p>The accepted values of these flags are:
+		COMMENT <ul>
+		COMMENT <li>{@link net.minecraft.block.Block#NOTIFY_ALL Block.NOTIFY_ALL}</li>
+		COMMENT <li>{@link net.minecraft.block.Block#NOTIFY_NEIGHBORS Block.NOTIFY_NEIGHBORS}</li>
+		COMMENT <li>{@link net.minecraft.block.Block#NOTIFY_LISTENERS Block.NOTIFY_LISTENERS}</li>
+		COMMENT <li>{@link net.minecraft.block.Block#NO_REDRAW Block.NO_REDRAW}</li>
+		COMMENT <li>{@link net.minecraft.block.Block#REDRAW_ON_MAIN_THREAD Block.REDRAW_ON_MAIN_THREAD}</li>
+		COMMENT <li>{@link net.minecraft.block.Block#FORCE_STATE Block.FORCE_STATE}</li>
+		COMMENT <li>{@link net.minecraft.block.Block#SKIP_DROPS Block.SKIP_DROPS}</li>
+		COMMENT <li>{@link net.minecraft.block.Block#MOVED Block.MOVED}</li>
+		COMMENT <li>{@link net.minecraft.block.Block#SKIP_LIGHTING_UPDATES Block.SKIP_LIGHTING_UPDATES}</li>
```

Comment:
Consider using `{@value net.minecraft.block.Block#NOTIFY_ALL} - "the thing they do"`. In javadoc, it renders with the actual value of that flag, like 1, 2, 4, 8, etc.

Reply:
No, I would rather encourage people to use the named constants than the literal values